---
layout: plain_cv
title: Victoria Cheung, PhD
---
# Victoria Cheung, PhD
Computational Biologist | Health Data Scientist

<div id="webaddress">
<a href="mailto:victoriakcheung@gmail.com"><i class="fas fa-regular fa-envelope"></i> victoriakcheung[at]gmail[dot]com</a>
</div>

<div id="webaddress">
<a href="https://www.linkedin.com/in/victoriacheung"><i class="fab fa-linkedin"></i> linked-in</a>
| <a href="https://scholar.google.com/citations?user=8DJJyEIAAAAJ&hl=en&authuser=1"><i class="fas fa-thin fa-user-graduate"></i> google scholar</a>
| <a href="https://vic-cheung.github.io/"><i class="fas fa-globe"></i> website</a>
| <a href="https://github.com/vic-cheung"><i class="fab fa-github"></i> github</a>
| <a href="https://twitter.com/victoriakcheung"><i class="fab fa-twitter"></i> twitter</a>
</div>

## Education

`2015-2021`
(__UCSF__) University of California, San Francisco,
__Ph.D.__ Genetics; concentration in Neuroscience

`2011-2015`
(__UCSD__) University of California, San Diego,
__B.S.__ Microbiology; minor: Chinese Studies

## Additional Courses

`2021`
__Genentech L.E.A.D Discovery Program__,
Certification. Supply Chain Management

`2017`
__Cold Spring Harbor Laboratory__,
Vision: _Linking Circuits, Perception, and Behavior_

## Experience

### Career-related

`APR 2022 - Present`
*[Freenome](https://www.freenome.com/about-f)*<br>
__Computational Biologist__<br>
-Apply bioinformatics, data science, and computational methods to analyze multi-omic data to reveal, model, and interpret changes in both the cancer (pathways, gene activities, proteins) and the immune system (composition, activity, and repertoires) associated with clinical outcomes, and generate new insights and interpretations.<br>
-Leverage existing computational methods and develop new ones to extract immunological signals from existing and new data.<br>
-Partner cross-functionally in the scientific planning and execution of collaborative projects, such as molecular and cancer biologists, immunologists, computational biologists, medical affairs, commercial, business development.<br>
-Execute research projects to model various biological changes resulting from diseases such as cancer, autoimmune disease, and infection with various business partners.<br>

___Technologies used:___<br>
-Python (ScanPy, NumPy, Pandas, scikit-learn, Matplotlib, SciPy, Freenome proprietary software)<br>
-R (fgsea, Seurat)<br>

<br><br>

`SEP 2021 - Present`
*[Genentech](https://www.gene.com/)*<br>
__Oncology Bioinformatics & Molecular Oncology PhD Intern__<br>
-Characterized gene signature development and refinement for T cell signaling pathways in cancer models<br>
-Wrote a data processing pipeline utilizing Scanpy, Numpy, Pandas, scikit-learn.<br>
-Performed statistical analyses on different drug treatment populations (gene set enrichment analysis, differential gene expression analysis).<br> 
-Utilized supervised batch correction techniques and unsupervised clustering algorithms (UMAP, topic modeling) to visualize and analyze single cell RNA seq data outputs.<br>
-Wrote custom plotting functions using Matplotlib to better visualize the effect of drug treatments.<br>

___Technologies used:___<br>
-Python (ScanPy, NumPy, pandas, scikit-learn, Matplotlib, SciPy)<br>
-R (fgsea, SingleCellExperiment, Seurat)<br>

<br><br>

`JUL 2016 - DEC 2021`
*[Evan Feinberg Lab](http://www.evanfeinberglab.com/who-we-are) @ [UCSF](https://tetrad.ucsf.edu/)*<br>
__Graduate Resarcher in Single-cell Omics, Systems Neuroscience__<br>
__Project 1__<br>
Developed a multiplexed, high-throughput, single-cell sequencing method for neurons that preserve connectivity information in addition to obtaining molecular identity (VECTORseq). Git Repo [here](https://github.com/vic-cheung/vectorseq).<br>
-Wrote the data processing pipeline using Python after genome alignment using Cellranger (10x Genomics) on an AWS EC2 instance.<br>
-Used unsupervised machine learning techniques such as t-SNE/UMAP clustering to match molecular identities to cellular function and role in behavioral output.<br>
-Implemented nearest neighbors algorithms to account for batch differences when merging datasets.<br>
-Streamlined brain dissociation techniques and increased neuron survivability yield 100-fold based on data-driven outcomes from clustering analyses.<br>
-Validated clustering results of single-cell sequencing against the [2020 10x sequencing dataset from the Allen Atlas](https://portal.brain-map.org/atlases-and-data/rnaseq/mouse-whole-cortex-and-hippocampus-10x) and that the methodology was functional.<br>
-Evaluated range of highly variable genes expressed per cluster for the validation of cell identity.<br>
-Managed collaborations with the Chan-Zuckerberg Biohub (Spyros Darmanis Group, now @ Genentech).<br>

___Technologies used:___<br>
 -AWS (EC2/S3)<br>
 -Linux, bash, CellRanger<br>
 -Python (ScanPy, NumPy, pandas, scikit-learn, Matplotlib, SciPy)<br>
 -FIJI, Zen<br>
 -Illumina Next Gen Sequencing, 10x Genomics 5' Sequencing<br>
 -stereotaxic surgeries, viral delivery<br>
<br>

__Project 2__<br>
Designed an audition-based behavioral paradigm to study sensorimotor integration in the context of mice.<br>
-Wrote custom software to support custom-built hardware using serial communication between MATLAB and an Arduino microprocessor, which increased productivity by - 6-fold from the parallelization and automation of data acquisition, storage, and analysis.<br>
-Used this system in exploring how sensory input is represented in the brain and transformed into behavioral commands, using mice as the model organism.<br>
-Wrote custom analyses software to automate, refine, and interpret both raw behavioral data and fiber photometry signals.<br>
-Used CAD software to design and 3D print custom behavioral apparatuses.<br>
-Refined surgical protocols to increase survival surgery success by 20%.<br>
-Delivery of viruses, drugs, and organic dyes into the mouse brain.<br>
-Performed physiology recordings on brain slices to validate optogenetic and fiber photometry experiments.<br>
-Assembled fiber photometry and optogenetic manipulation equipment to record and perturb neuronal activity in the context of quantitative behavioral assays.<br>

___Technologies used:___<br>
-CAD Software (Onshape, Cura, eMachineShop)<br>
-MATLAB<br>
-Arduino (Uno)<br>
-FIJI, Zen<br>
-stereotaxic surgeries, viral delivery, fiber optic implants<br>
-fiber photometry, optogenetics<br>
-immunohistochemistry<br>

<br><br>

`MAY 2020 - JUL 2020`
*[Insight Data Science](https://insightfellows.com/health-data) @Silicon Valley*<br>
__Health Data Science Fellow__<br>
<p>Developed a predictive clinical calculator to assess Acute Kidney Injury in hospitalized patients, which would result in better management, care/medication dosing, injury prevention, and reduced hospital length of stay, thus freeing up occupied resources and minimizing financial costs to both patient and hospital.</p>
-Utilized PostgreSQL querying to gather relevant data from the MIMIC-III database and manipulated the data with Python Pandas from 25 tables of data, 46,000 patients, thousands of diagnoses and lab tests, and clinical documentation-- generating over 3 million rows of data and 70 unique features comprising lab tests and demographic information.<br>
-Used supervised machine learning in Python such as regression models from scikit-learn and XGBoost to forecast Acute Kidney Injury, with a predictive accuracy of ~91%.<br>
-Medium Article in Towards Data Science: [Predicting Acute Kidney Injury in Hospitalized Patients Using Machine Learning](https://towardsdatascience.com/predicting-acute-kidney-injury-in-hospitalized-patients-53ca07525e67).<br>

___Technologies used:___<br>
-Python (NumPy, pandas, scikit-learn, Matplotlib, SciPy, XGBoost)<br>
- SQL
-AWS (EC2, S3, Route 53)<br>
-Streamlit<br>

### Other

`2013-2018`
_[Genentech](https://www.gene.com/)_<br>
__[Genentech] Discovery Program L.E.A.D Supply Chain__<br>
_[Certification](https://docs.google.com/viewer?url=https://github.com/vic-cheung/vic-cheung.github.io/raw/main/pdfs/VictoriaCheung_SupplyChainCertificate.pdf)_<br>
-Learned about the fundamentals of supply chain, how the supply chain spans a variety of roles throughout Genentech's delivery of therapies as well as its involvement in providing medication access to underserved communities and its drive towards sustainability.<br>
-Chatted with supply chain business leaders to interact with individuals in the industry.<br>
-Discussed the transferability of skills from the PhD to business/supply chain.<br>
-Participated as Operations Lead in a supply chain simulation where my team and I placed second overall.<br>

<br><br>

`APR 2016 - JUN 2016`
_[Guo Huang Lab](https://www.cvri.ucsf.edu/~huang/lab/Research.html) @ UCSF_<br>
__Graduate Researcher--rotation__<br>
Area of Research: Regenerative Potential of Cardiomyocytes<br>
-Performed heart explants for cell culture and subsequent drug studies for the purpose of exploring organ regeneration and repair in neonatal miceâ€”with an emphasis on the pathways that regulate resident stem cell activation and mature cell de-differentiation/proliferation.<br>
-Explored organ regeneration and development from an evolutionary standpoint across different species of animals i.e. naked mole rat, finch, rat, mouse, zebrafish.<br>
-Utilized innovative and integrated approaches in engineering, single cell analysis, advanced imaging microscopy, drug delivery, and genome manipulation technology.<br>

<br><br>

`SEP 2015 - DEC 2015`
_[Dengke Ma Lab](https://malab.ucsf.edu/research) @ UCSF_<br>
__Graduate Researcher--rotation__<br>
Area of Research: Homeostatic Response to Extreme Abiotic Factors<br>
-Created a functional mutant line in C. elegans via cDNA microinjections and exposed the mutants to extreme abiotic environments via behavioral assays to understand cellular intrinsic tolerance of hypoxia/anoxia and hypothermia.<br>
-Utilized RNA-seq to identify genes implicated in cryopreservation/hypoxia-tolerance with therapeutic potential.<br>
-Obtained qualitative behavioral data on how animals sensed and responded to changes in internal states to elicit behavior and maintain homeostasis.<br>

<br><br>

`SEP 2012 - JUL 2015`
_[Andrew D. Huberman Lab](https://hubermanlab.com/about/) @ UCSD_<br>
__Reasearch Fellowship: [UCLEADS](https://www.ucop.edu/graduate-studies/initiatives-outreach/uc-leads.html) & [STARS](https://grad.ucsd.edu/diversity/programs/stars/index.html)__<br>
Area of Research: Binocular Plasticity & Dynamic Strategy Implementation in Cuttlefish <br>
-Developed a model of visual perception and prey capture using cuttlefish to study the neural circuit organization supporting flexible eye movements. Underlying goal: provide insight into amblyopia (lazy eye).<br>
-Optimized behavioral parameters and refined surgical techniques for the novel model organism.<br>
-Increased consistency between experiments for reproducibility.<br>
-Streamlined tracking and analysis of dynamical eye movements using multi-planar high-speed imaging and Simi Motion software and increased productivity and output by 40%.<br>
-3D-reconstructed neuron structure for morphometric analysis using Neurolucida.<br>

<br><br>

`JUN 2014 - SEP 2014`
_[David R. Copenhagen Lab](https://neurograd.ucsf.edu/people/david-copenhagen-phd) @ UCSF_<br>
__Research Fellowship: [UCSF SRTP](https://graduate.ucsf.edu/srtp)__<br>
Area of Research: Effects of Light Dependent Ca<sup>2+</sup> Signaling during retinal development<br>
-Explored the effects of light dependent Ca<sup>2+</sup> waves in the developing mouse retina with a focus on the coupling of amacrine cells to melanopsin ganglion cells via gap junctions.<br>
-Characterized appropriate transgenic lines and established baseline comparisons in adult retina to observe and document deviations from the developed animal to the developing animal.<br>
-Performed retinal dissections for cell coupling studies.<br>

## Mentorship & Diversity

`JUN 2026 - SEP 2021`
__Mentor for Undergraduates__<br>
-Trained and mentored 3 undergraduates on performing research tasks on how to: think independently, plan experiments, perform surgical protocols, and analyze data. Gave career/research advice.<br>
-Post-graduation outcomes of the 3 undergraduates:<br>
(1) data analyst @[BoxLunch](https://www.boxlunch.com/homepage?cm_mmc=SEM-_-GGL-_-BR-_-DMY-_-541787926_57911524309&gclid=CjwKCAiAx8KQBhAGEiwAD3EiP-FK2iMX9GHn7rNmiIQvYcnjXlKJkunprrDSj0u4bnXOVlWWHPe9YBoC1TMQAvD_BwE)<br>
 (2) research scientist @[Alkahest](https://www.alkahest.com/)<br>
 (3) applying to medical school<br>

<br><br>

`JUN 2019 - AUG 2019`
_[UCSF SRTP](https://graduate.ucsf.edu/srtp)_<br>
__Student Advisor__,
Developed curriculum for and taught curriculum to teach rising junior and senior undergraduates on:<br>
 (1) how to become a strong graduate school applicant<br>
 (2) how to create compelling posters and presentations<br>
 (3) how to write personal statements<br>
 (4) how to read and dissect scientific papers.<br>

<br><br>

`JAN 2016 - JUN 2016`
_[UCSF Science and Health Education Partnership](https://sep.ucsf.edu/what-we-do/)_<br>
__Student Teacher__,
-Created and developed a series of interactive and investigative lesson plans to teach freshman biology.<br>
-Mentored URMs and socioeconomically disadvantaged students on different career paths in science.<br>

<br><br>

`MAR 2013 - JUN 2015`
_[UCSD](https://www.ucop.edu/graduate-studies/initiatives-outreach/uc-leads.html)_<br>
__UC Leadership Through Advanced Degrees Scholar (UCLEADS)__,
"Mentorship program for underprivileged and socioeconomically disadvantaged undergraduates for success in graduate school to later assume positions of leadership in industry, government, public service, and academia following completion of a doctoral STEM degree"<br>
-Two-way avenue:<br>
 (1) Received mentorship from prior two cohorts as part of the incoming cohort<br>
 (2) Provided mentorship to the next two cohorts while progressing through the program<br>

## Publications

__Cheung, V.__, Chung, P., and Feinberg, E.H. (2022) [Transcriptional profiling of mouse projection neurons with VECTORseq](https://doi.org/10.1016/j.xpro.2022.101625) STAR Protocols, 3(3):101625<br>

__Cheung, V.__, Chung, P., Bjorni, M., Shvareva, V.A., Lopez, Y.C., and Feinberg, E.H. (2021) [Virally Encoded Connectivity Transgenic Overlay RNA sequencing (VECTORseq) defines projection neurons involved in sensorimotor integration.](https://doi.org/10.1016/j.celrep.2021.110131) Cell Reports, 37(12):110131<br>

__Cheung, V.__ "[Predicting Acute Kidney Injury in Hospitalized Patients Using Machine Learning](https://towardsdatascience.com/predicting-acute-kidney-injury-in-hospitalized-patients-53ca07525e67)" Towards Data Science. Medium, 20 Jun. 2020. Web.<br>


## Conferences & Talks

### Scientific

`2021`
UCSF S.O.L.V.E. Health Tech: Digital Health Equity Summit | __Attendee__

`2021`
COSYNE (computational and systems neuroscience) | __Attendee__

`2018`
UCSF Tetrad | __Presenter__, _15 min talk_

`2017`
SFN Annual Conference | __Attendee__

`2017`
UCSF Tetrad | __Presenter__, _poster presentation_

`2016`
UC LEADs Research Symposium | __Attendee__

`2015`
UC LEADs Research Symposium | __Presenter__, _poster presentation_

`2014`
SFN Annual Conference | __Presenter__, _poster presentation_

`2014`
UCSF Summer Research Training Program Symposium | __Presenter__, _poster presentation + 15 min talk_

`2014`
UCSD Academic Enrichment Program | __Presenter__, _poster presentation_

`2014`
SACNAS National Conference | __Presenter__, _poster presentation_

`2014`
UCLEADs Annual Research Symposium | __Presenter__, _poster presentation_

`2013`
UCSD Academic Enrichment Program | __Presenter__, _poster presentation_

`2013`
SACNAS National Conference | __Presenter__, _poster presentation_

`2013`
UCSD STARS Summer Research Conference | __Presenter__, _15 min talk_

### Diversity & Outreach

`2022`
SRTP Co-Curriculum: What Can You Do With a PhD? | __Panelist__ (1hr)

`2019`
Northern California Forum for Diversity in Graduate Education | __Panelist__ (1hr)

`2018`
Northern California Forum for Diversity in Graduate Education | __Panelist__ (1hr)

## Honors & Awards
`2022` UCSF Diversity Graduation, Graduate Division Speaker

`2017` Helmsley Scholar

`2015` UC LEADs Symposium Presentation Award 

`2014` SACNAS National Research Conference Travel Scholarship 

`2013` UCSD STARS Scholarly Presentation Award

`2013` SACNAS National Research Conference Travel Scholarship 

`2013` UCSD Provost Honors

`2012` UCSD Provost Honors

`2012` Kaiser Permanente Valuable Volunteer Award 

`2011` UCSD Provost Honors 

`2011` Kaiser Permanente Student Achievement Award 


## Media Features

`2019`
__UCSF Poster__, [First Generation to College](https://docs.google.com/viewer?url=https://github.com/vic-cheung/vic-cheung.github.io/raw/main/pdfs/VictoriaCheung_FG2Cposter.pdf)

`2019`
__UCSF Article__, [Students Who Are First in Their Family to Attend College Share Stories, Experiences](https://www.ucsf.edu/news/2019/08/415181/students-who-are-first-their-family-attend-college-share-stories-experiences)


<br/><br/>
<p><small>&mdash; Last updated: Sep 20, 2022</small></p>
